static/src/inline-svgs/icon/star.svg

```diff
@@ -1 +1 @@
-<svg width="14" height="13"><path d="M0 5.2l3.7 2.8-1.4 4.6.5.4 3.7-2.8 3.7 2.8.5-.4-1.4-4.6 3.7-2.8-.2-.6h-4.6l-1.4-4.6h-.6l-1.4 4.6h-4.6l-.2.6z"/></svg>
+<svg width="14" height="13" viewBox="0 0 14 13"><path d="M0 5.2l3.7 2.8-1.4 4.6.5.4 3.7-2.8 3.7 2.8.5-.4-1.4-4.6 3.7-2.8-.2-.6h-4.6l-1.4-4.6h-.6l-1.4 4.6h-4.6l-.2.6z"/></svg>
```

Comment:
It probably could, but not without refactoring all the stars across the site. So I'm going to suggest that's part of another PR instead...